.github/workflows/terraform-github.yml

```diff
@@ -24,28 +24,65 @@ jobs:
   github-plan-and-apply:
     runs-on: ubuntu-latest
     env:
-      AWS_ACCESS_KEY_ID:  ${{ secrets.AWS_ACCESS_KEY_ID }}
-      AWS_SECRET_ACCESS_KEY:  ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       TF_VAR_github_token: ${{ secrets.TERRAFORM_GITHUB_TOKEN }}
       TF_IN_AUTOMATION: true
+      AWS_REGION: "eu-west-2"
+  ENVIRONMENT_MANAGEMENT: ${{ secrets.MODERNISATION_PLATFORM_ENVIRONMENTS }}
```

Comment:
this line fails the pipeline actually; it's the indentation that needs fixing

Reply:
Done